### PR TITLE
Task DEV-1

### PR DESCRIPTION
**Pull request by MaryVazhnik**

### CODE REVIEW

File: DEV_1/DEV_1/FindSymbol.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        /// <param name="args"></param>
+        static void Main(string[] args)
+        {
+            string str = args[0];
+            int maxcount = 0;
+            int count = 1;
+            for (int n = 0; n < str.Length - 1; n++)
+            {             
+                    if (str[n] == str[n + 1])
+                        count++;
```
Comment (IgorKheidorov) [inline DEV_1/DEV_1/FindSymbol.cs]: Code style

File: DEV_1/DEV_1/FindSymbol.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+                    else if (count > maxcount) 
+                    {
+                        maxcount = count;
+                        count = 1; 
+                    }    
+            }
+
+            if (count > maxcount)
```
Comment (IgorKheidorov) [inline DEV_1/DEV_1/FindSymbol.cs]: Code style

File: DEV_1/DEV_1/FindSymbol.cs [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+                maxcount = count;
+            Console.WriteLine("Amount: " + maxcount);
+            Console.ReadKey();
```
Comment (IgorKheidorov) [inline DEV_1/DEV_1/FindSymbol.cs]: Never use ReadKey() only to stop application, forbidden

File: DEV_1/DEV_1/FindSymbol.cs [hunk 4]
```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
```
Comment (IgorKheidorov) [inline DEV_1/DEV_1/FindSymbol.cs]: Remove unused usings

File: DEV_1/DEV_1/FindSymbol.cs [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading;
+
+namespace DEV_1
+{
+    /// <summary>
+    /// This class contains a method that works with a string
+    /// </summary>
+    class FindSymbol
```
Comment (IgorKheidorov) [inline DEV_1/DEV_1/FindSymbol.cs]: Bad name for class

File: DEV_1/DEV_1/FindSymbol.cs [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+            }
+
+            if (count > maxcount)
+                maxcount = count;
+            Console.WriteLine("Amount: " + maxcount);
+            Console.ReadKey();
+        }
+    }
+}
+
```
Comment (IgorKheidorov) [inline DEV_1/DEV_1/FindSymbol.cs]: Empty lines

**Review by IgorKheidorov — CHANGES_REQUESTED** [review]

File: DEV_1/DEV_1/DEV_1.csproj.user [hunk 7]
```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (AlexanderSoroka) [inline DEV_1/DEV_1/DEV_1.csproj.user]: User-related files must not be commited, remove it from review

File: DEV_1/DEV_1/Properties/AssemblyInfo.cs [hunk 8]
```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// Управление общими сведениями о сборке осуществляется с помощью
```
Comment (AlexanderSoroka) [inline DEV_1/DEV_1/Properties/AssemblyInfo.cs:5]: Do not use Russian

File: DEV_1/DEV_1/RepeatSequence.cs [hunk 9]
```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Text;
+
+namespace DEV_1
+{
+    /// <summary>
+    /// This class contains a method that works with a string
+    /// </summary>
+    class RepeatSequence
```
Comment (AlexanderSoroka) [inline DEV_1/DEV_1/RepeatSequence.cs]: class name should be noun

File: DEV_1/DEV_1/RepeatSequence.cs [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+                    count++;
+                }
+                else if (count > maxcount)
+                {
+                    maxcount = count;
+                    count = 1;
+                }    
+            }
+            if (count > maxcount)
+                maxcount = count;
```
Comment (AlexanderSoroka) [inline DEV_1/DEV_1/RepeatSequence.cs]: codestyle violation

File: DEV_1/DEV_1/RepeatSequence.cs [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+            Console.WriteLine("Amount: " + maxcount);
+            Console.ReadLine();
```
Comment (AlexanderSoroka) [inline DEV_1/DEV_1/RepeatSequence.cs]: Useless readline

**Review by AlexanderSoroka — CHANGES_REQUESTED** [review]

**Review by AlexanderSoroka — APPROVED** [review]
Not accepted as some comments were ignored

**Pull request merged**